charts/geonode/README.md

```diff
@@ -193,9 +193,24 @@ Helm Chart for Geonode a web-based application and platform for developing geosp
pename\": \"csw_typename\",\n      \"pycsw:Schema\": \"csw_schema\",\n      \"pycsw:MdSource\": \"csw_mdsource\",\n      \"pycsw:InsertDate\": \"csw_insert_date\",\n      \"pycsw:XML\": \"metadata_xml\",\n      \"pycsw:AnyText\": \"csw_anytext\",\n      \"pycsw:Language\": \"language\",\n      \"pycsw:Title\": \"title\",\n      \"pycsw:Abstract\": \"raw_abstract\",\n      \"pycsw:Keywords\": \"keyword_csv\",\n      \"pycsw:KeywordType\": \"keywordstype\",\n      \"pycsw:Format\": \"spatial_representation_type_string\",\n      \"pycsw:Source\": \"source\",\n      \"pycsw:Date\": \"date\",\n      \"pycsw:Modified\": \"date\",\n      \"pycsw:Type\": \"csw_type\",\n      \"pycsw:BoundingBox\": \"csw_wkt_geometry\",\n      \"pycsw:CRS\": \"csw_crs\",\n      \"pycsw:AlternateTitle\": \"alternate\",\n      \"pycsw:RevisionDate\": \"date\",\n      \"pycsw:CreationDate\": \"date\",\n      \"pycsw:PublicationDate\": \"date\",\n      \"pycsw:Organization\": \"organizationname\",\n      \"pycsw:OrganizationName\": \"organizationname\",\n      \"pycsw:SecurityConstraints\": \"securityconstraints\",\n      \"pycsw:ParentIdentifier\": \"parentidentifier\",\n      \"pycsw:TopicCategory\": \"topiccategory\",\n      \"pycsw:ResourceLanguage\": \"language\",\n      \"pycsw:GeographicDescriptionCode\": \"geodescode\",\n      \"pycsw:Denominator\": \"denominator\",\n      \"pycsw:DistanceValue\": \"distancevalue\",\n      \"pycsw:DistanceUOM\": \"distanceuom\",\n      \"pycsw:TempExtent_begin\": \"temporal_extent_start\",\n      \"pycsw:TempExtent_end\": \"temporal_extent_end\",\n      \"pycsw:ServiceType\": \"servicetype\",\n      \"pycsw:ServiceTypeVersion\": \"servicetypeversion\",\n      \"pycsw:Operation\": \"operation\",\n      \"pycsw:CouplingType\": \"couplingtype\",\n      \"pycsw:OperatesOn\": \"operateson\",\n      \"pycsw:OperatesOnIdentifier\": \"operatesonidentifier\",\n      \"pycsw:OperatesOnName\": \"operatesoname\",\n      \"pycsw:Degree\": \"degree\",\n      \"pycsw:AccessConstraints\": \"restriction_code\",\n      \"pycsw:OtherConstraints\": \"raw_constraints_other\",\n      \"pycsw:Classification\": \"classification\",\n      \"pycsw:ConditionApplyingToAccessAndUse\": \"conditionapplyingtoaccessanduse\",\n      \"pycsw:Lineage\": \"lineage\",\n      \"pycsw:ResponsiblePartyRole\": \"responsiblepartyrole\",\n      \"pycsw:SpecificationTitle\": \"specificationtitle\",\n      \"pycsw:SpecificationDate\": \"specificationdate\",\n      \"pycsw:SpecificationDateType\": \"specificationdatetype\",\n      \"pycsw:Creator\": \"creator\",\n      \"pycsw:Publisher\": \"publisher\",\n      \"pycsw:Contributor\": \"contributor\",\n      \"pycsw:Relation\": \"relation\",\n      \"pycsw:Links\": \"download_links\",\n  },\n}","pod_name":"pysw","port":8000,"replicaCount":1,"resources":{"limits":{"cpu":"500m","memory":"1Gi"},"requests":{"cpu":"500m","memory":"1Gi"}}}` | pycsw integration is based on https://github.com/geopython/pycsw/blob/master/docker/kubernetes |
```

Comment:
I would not bloat the README so much. In doubt, just [remove documentation](https://github.com/zalf-rdm/geonode-k8s/blob/4c688f745491cbb8db1a154a5bb2a1c85be18ebf/charts/geonode/values.yaml#L371) for the whole `pycsw` part.